components/world_news_api/sources/news-event/news-event.mjs

```diff
@@ -0,0 +1,165 @@
+import app from "../../world_news_api.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  name: "New News",
+  description: "Emit new event when a new news is fetched. Calling this endpoint requires 1 point per page, up to 1000 news.",
+  key: "world_news_api-news-event",
```

Comment:
Please refer to this docs on [component key patterns](https://pipedream.com/docs/components/guidelines/#component-key-pattern)

Reply:
Not sure if I get it